asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/sandbox.cypher

```diff
@@ -1,13 +1,8 @@
 CALL gds.graph.project(
   // Name of the projection
-  'user-rated-movie',
+  ,
   // Labels of nodes to include in the projection
-  [
-    /* ... */
-  ],
+  [],
   // Relationship types to include in the projection
-  {
-    /* ... */
-  }
-);
-
+  []
+)
```

Comment:
I like the comments as placeholders here - or replicate the `????` from #196 for consistency